MobileFilter/sections/specific_web.txt

```diff
@@ -75,6 +75,7 @@ otakei.otakuma.net,nikkan-gendai.com,ranking.goo.ne.jp,m.hellporno.com,topics.sm
 !
 ! SECTION: Regular rules
 !
+cars.av.by##.promo-app
```

Comment:
If I understand correctly, it's a banner with a link to their own app, so I think that it would be better to add it to - https://github.com/AdguardTeam/AdguardFilters/blob/master/AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt

Reply:
That's right, I moved it